docs/src/lints/roblox_complex_udim2_new.md

```diff
@@ -0,0 +1,14 @@
+# roblox_complex_udim2_new
```

Comment:
Is calling it complex the right naming? The way I see it is the args are simple to where `fromScale`/`fromOffset` should be used instead.

Reply:
Not sure! cc @Kampfkarren for input. This is the best we could think of!

Reply:
Clippy tends to call these "manual_xxx", but I can't think of a good way to use that pattern when this lint (correctly, IMO) covers both fromScale or fromOffset. Perhaps `manual_fromscale_or_fromoffset`? 🤷‍♀️

Reply:
Clippy tends to call these "manual_xxx", but I can't think of a good way to use that pattern when this lint (correctly, IMO) covers both fromScale or fromOffset. Perhaps `manual_fromscale_or_fromoffset`? 🤷‍♀️

Reply:
`manual_udim2_zeros` maybe?

Reply:
`manual_udim2_zeros` sounds nicer, but clippy names the xxx in "manual_xxx" to be the thing that the code should do than what it's currently doing. For ex <https://rust-lang.github.io/rust-clippy/master/#/manual_assert>  is `manual_assert` rather than `manual_if_then_panic`. If we follow that pattern then `manual_fromscale_or_fromoffset` would be more consistent with what the code should do than what it's currently doing.

Reply:
I like `manual_fromscale_or_fromoffset`

Reply:
Updated!